content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md

```diff
@@ -0,0 +1,96 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/api/core/v1"
+  kind: "ObjectReference"
+content_type: "api_reference"
+description: "ObjectReference 包含足够的信息，可以让您检查或修改引用的对象。"
```

Comment:
Please attach the English content for the frontmatter, as comment after line 12.